app/scripts/components/common/catalog/prepare-datasets.ts

```diff
@@ -21,7 +23,7 @@ export function prepareDatasets (
   data: DatasetData[] | StoryData[],
   options: FilterOptionsType
 ) {
-  const { sortField, sortDir, search, taxonomies, filterLayers } = options;
+  const { sortField = 'name', sortDir, search, taxonomies, filterLayers } = options;
```

Comment:
we should really type `sortField` but unsure of all strings it could be, should default to 'name' though